azavea/raster-foundry
# Reset heartbeat when resetting layer.

Connects #319 

When we reset a layer we also need to reset the heartbeat status. This field
is used to note that a failed EMR heartbeat was returned and we rely on
knowing that two failed heartbeats were returned to decide if the cluster
actually failed to complete the job.

Also adds the heartbeat status to the admin UI.

### To Test
For speed of testing, this can be done manually by changing the layer status fields in the admin interface.
 * Start a job.
 * Ensure the job has failed and has a timestamp for the status_heartbeat field.
 * In the UI "retry" the job. 
 * Check to see that the heartbeat status has been nulled out.
